restful.go

```diff
@@ -558,3 +560,16 @@ func (c *APIClient) UploadToStageByAPI(stage *StageLocation, input *bufio.Reader
 
 	return nil
 }
+
+func readVersionFile() string {
+	// 读取 VERSION 文件
+	data, err := os.ReadFile("VERSION")
```

Comment:
`VERSION` file is not available at runtime.

Reply:
Oh, Forgot it.

Reply:
Fixed by adding a `version.go` file.